regression/goto-analyzer/sensitivity-function-call-array/main.c

```diff
@@ -0,0 +1,32 @@
+#include <assert.h>
+
+int *bar(int *arr_unmodified, int *arr_modified);
+
+int main(int argc, char *argv[])
+{
+  int arr_x[] = {1, 2, 3, 4};
+  int arr_y[] = {1, 2, 3, 4};
+  int *p2arr = arr_x;
+
+  p2arr = bar(arr_x, arr_y);
+  assert(*arr_y==2);
+  // assert(arr_y[1]==3);
+
+  assert(p2arr==arr_y);
+  assert(*p2arr==2);
```

Comment:
These asserts probably need to become `__CPROVER_assert` to work on Alpine/Windows